### ISSUE TITLE
0.6 Billion parameters

### ISSUE THREAD

**Issue by ashrafbay** [NVlabs/Sana#8]
The 1.6 model is very good and I like it and it easily outperforms the larger models. I was wondering if it is possible to put a demo for the 0.6 model because I am very interested in it as well. It will solve many problems in computing if it is good or competes with the larger models. Thanks

**Comment by bil-ash** [comment]
Yes, a demo of the 0.6b model along with the weights of the 0.6b model would be much appreciated.

**Comment by ashrafbay** [comment]
> نعم، سيكون من دواعي سرورنا الحصول على عرض توضيحي للنموذج 0.6b مع أوزان النموذج 0.6b.

That would be great. You have done an amazing job and I thank you.

**Comment by lawrence-cj** [comment]
We will try our best to release all the models.

**Comment by TomLucidor** [comment]
As long as every model size up to 8, 12, and 16GB worth is released, it would be good (not sure if LLM or image weight quantization would help or not tho)

**Comment by nitinmukesh** [comment]
> We will try our best to release all the models.

Thank you @lawrence-cj .
I am sure, we less VRAM users will not feel left out. :)

**Comment by bil-ash** [comment]
@lawrence-cj When will the 0.6b model weight be released?

**Comment by lawrence-cj** [comment]
As you can see. We are preparing several models at the same time. I'll try my best to release them all ASAP.
@bil-ash

**Comment by AfterHAL** [comment]
> As you can see [here](https://huggingface.co/collections/Efficient-Large-Model/sana-673efba2a57ed99843f11f9e). We are preparing several models at the same time. I'll try my best to release them all ASAP. @bil-ash

Yes. Thanks a lot to the team for this great job.

**Comment by yujincheng08** [comment]
https://huggingface.co/Efficient-Large-Model/Sana_600M_512px
https://huggingface.co/Efficient-Large-Model/Sana_600M_1024px
